utils/exec.go

```diff
@@ -135,3 +147,97 @@ func RunCmd(command string) (output string, exitCode int, startedAt time.Time, f
 	log.Infof("Exit Code: %d", exitCode)
 	return
 }
+
+func RunContinuousReportRun(fn func(chunk string) error, cmdArg string, thresholds Thresholds) (int, error) {
```

Comment:
Not sure the utils package is the best place for this function, which is very specific to the continuous report run command...

Reply:
yes, it previously was out of this location. But the idea for this is about the possibility of future uses; i mean, a continuous command but with different management for chunked responses.